FINS/FINS-IOC-01App/Db/sans2d-pump-add.template

```diff
@@ -0,0 +1,79 @@
+# $(ADDR).02	           Pump $(PUMP_NUM) status            Emergency stop button - OK
+# $(ADDR).03	           Pump $(PUMP_NUM) status            SP Guard Warning
+# $(ADDR).04	           Pump $(PUMP_NUM) status            SP Guard Fault
+
+# FULL_PREFIX = Instrument and IOC prefix
+
+record(bi, "$(FULL_PREFIX)PUMP$(PUMP_NUM)_STAT:STOP_OK") {
+	field(DESC, "Pump $(PUMP_NUM) emergency stop OK")
+	field(DTYP, "asynInt32")
+	field(INP,  "@asyn(PLC, $(ADDR), 5.0) FINS_DM_READ_B2")
+	field(ZNAM, "STOP BAD")
+	field(ONAM, "STOP OK")
+	field(SCAN, "1 second")
+	field(SIML, "$(FULL_PREFIX)SIM")
+    field(SIOL, "$(FULL_PREFIX)SIM:PUMP$(PUMP_NUM)_STAT:STOP_OK")
+    field(SDIS, "$(FULL_PREFIX)DISABLE")
+	field(FLNK, "$(FULL_PREFIX)PUMP$(PUMP_NUM)_STAT:STOP_BAD.PROC")
+    info(INTEREST, "LOW")
+	field(ZSV, "MAJOR")
+	info(alarm, "Vacuum")
+}
+
+record(calc, "$(FULL_PREFIX)PUMP$(PUMP_NUM)_STAT:STOP_BAD") {
+	field(DESC, "Pump $(PUMP_NUM) emergency stop bad")
+	field(INPA,  "$(FULL_PREFIX)PUMP$(PUMP_NUM)_STAT:STOP_OK MS")
+	field(CALC, "!A")
+	field(ASG, "READONLY")
+    info(INTEREST, "HIGH")
+}
+
+record(bi, "$(FULL_PREFIX)PUMP$(PUMP_NUM)_STAT:GUARD_WARN") {
+	field(DESC, "Pump $(PUMP_NUM) SP guard warning")
+	field(DTYP, "asynInt32")
+	field(INP,  "@asyn(PLC, $(ADDR), 5.0) FINS_DM_READ_B3")
```

Comment:
I don't think there's a performance concern, we're hitting the PLC much less than it could take in theory. I can read the whole block if you think it's better for consistency? I did it one bit at a time here as I only needed a few bits.